pkg/security/tests/open_test.go

```diff
@@ -175,6 +175,51 @@ func TestOpen(t *testing.T) {
 		})
 	})
 
+	t.Run("ftruncate", func(t *testing.T) {
+		SkipIfNotAvailable(t)
+
+		defer os.Remove(testFile)
+
+		var f *os.File
+		test.WaitSignal(t, func() error {
```

Comment:
why do we need to call waitsignal here if we don't check it?

Reply:
I'm not sure I think this is just an artifact of this being copied from the truncate test, will do a clean up of this in a follow up PR

Reply:
thanks!